docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -641,6 +641,23 @@ public Consumer<Message<String>> consume2() {
     };
 }
 ----
+
+===== Resource Provision
+EventHub and ConsumerGroup could be auto create by add such properties:
```

Comment:
Let's write it as "Event Hubs binder supports provisioning of event hub and consumer group, users could use below properties to enable provisioning."